### ISSUE TITLE
Display Bitcoin confirmations received/remaining on Order summary page

### ISSUE THREAD

**Issue by alahj** [kimitzu/kimitzu-client#80]
On the order summary page, there is no indication that a payment has been made while the Bitcoin confirmations are processing.  This can be unnerving to the user, as they wonder if their payment has been lost due to a bug in the code.   

1.  Display a note to the user that the payment won't be complete until there have been (3? 4?) confirmations of the transactions on the Bitcoin network.  And display the current number of confirmations, so that they can see that something is happening.  Let the user know that it might be several minutes or even hours if the network is busy.  

2.  Once the bitcoin transaction has been started, are they really able to cancel the order?   If they can't actually cancel it, that button should not be displayed.  

3.  Likewise on the seller side, there should be note that the seller has received the bitcoin (provisionally), pending sufficient confirmations from the network.  Right now, it says that the buyer hasn't sent payment, even though they have.  

![Screen Shot 2020-01-23 at 3 17 33 AM](https://user-images.githubusercontent.com/53591309/72980228-044d1680-3d8f-11ea-8c91-86ed07884fc0.png)

**Comment by alahj** [comment]
![Screen Shot 2020-01-23 at 3 11 50 AM](https://user-images.githubusercontent.com/53591309/72980589-bdabec00-3d8f-11ea-8a1a-34da414142c2.png)

**Comment by 24thsaint** [comment]
Yes, they should be able to cancel the order if the vendor was offline at the time of transaction.
